### PR TITLE
add(epp) : prerequisites for Azure monitoring in docv2 format

### PR DESCRIPTION
**Pull request by lchrdn**
## Description

Adding a new documentation describing how to get tokens for setting up Azure monitoring.

This procedure aims to be part of each "Prerequisites" part of any Azure Plugin Pack documentation to ease update management.

[DOCV1 previous PR that was approved ](https://github.com/centreon/centreon-documentation/pull/1293)

## Target version

- [x] 20.10.x
- [x] 21.04.x
- [x] 21.10.x
- [x] 22.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+* Azure CLI ('azcli').
+
+Centreon recommande d'utiliser la méthode **API** plutôt que Azure CLI pour les raisons suivantes :    
+* L'API est plus efficiente car elle n'implique pas d'exécution du programme azcli
+* L'API supporte l'authentification via une Application tandis que azcli pas pour l'instant
+
+> La découverte d'hôtes n'est disponible qu'en utilisant la méthode API.
+
+Pour plus d'information, rendez-vous sur la documentation Centreon :
+[Découverte d'hôtes](../../../monitoring/discovery/hosts-discovery.md)
```
Comment (itoussies) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/integrations/plugin-packs/tutorials/azure-credential-configuration.md:16]:
```suggestion
[Découverte d'hôtes](../../../monitoring/discovery/hosts-discovery)
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+* Azure CLI ('azcli').
+
+Centreon recommande d'utiliser la méthode **API** plutôt que Azure CLI pour les raisons suivantes :    
+* L'API est plus efficiente car elle n'implique pas d'exécution du programme azcli
+* L'API supporte l'authentification via une Application tandis que azcli pas pour l'instant
+
+> La découverte d'hôtes n'est disponible qu'en utilisant la méthode API.
+
+Pour plus d'information, rendez-vous sur la documentation Centreon :
+[Découverte d'hôtes](../../../monitoring/discovery/hosts-discovery.md)
```
Comment (itoussies) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/integrations/plugin-packs/tutorials/azure-credential-configuration.md:16]:
```suggestion
[Découverte d'hôtes](../../../monitoring/discovery/hosts-discovery)
```

File: versioned_docs/version-20.04/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+* Azure CLI ('azcli').
+
+Centreon recommends to use Azure Monitor API instead of Azure CLI for the following reasons: 
+* The API is much more efficient by avoiding CLI binary execution
+* The API supports application authentication while CLI does not (yet)
+
+> This discovery feature is only compatible with Azure Monitor API.
+
+More information about the Host Discovery module is available in the Centreon documentation:
+[Host Discovery](../../../monitoring/discovery/hosts-discovery.md)
```
Comment (itoussies) [inline versioned_docs/version-20.04/integrations/plugin-packs/tutorials/azure-credential-configuration.md:16]:
```suggestion
[Host Discovery](../../../monitoring/discovery/hosts-discovery)
```

File: versioned_docs/version-21.04/integrations/plugin-packs/tutorials/azure-credential-configuration.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+* Azure CLI ('azcli').
+
+Centreon recommends to use Azure Monitor API instead of Azure CLI for the following reasons: 
+* The API is much more efficient by avoiding CLI binary execution
+* The API supports application authentication while CLI does not (yet)
+
+> This discovery feature is only compatible with Azure Monitor API.
+
+More information about the Host Discovery module is available in the Centreon documentation:
+[Host Discovery](../../../monitoring/discovery/hosts-discovery.md)
```
Comment (itoussies) [inline versioned_docs/version-21.04/integrations/plugin-packs/tutorials/azure-credential-configuration.md:16]:
```suggestion
[Host Discovery](../../../monitoring/discovery/hosts-discovery)
```

**Review by itoussies — CHANGES_REQUESTED** [review]

**Review by itoussies — APPROVED** [review]

**Review by Sims24 — APPROVED** [review]

**Comment by lchrdn** [comment]
Outdated PR from before PP migration.
New PR https://github.com/centreon/centreon-documentation/pull/1425

**Pull request closed without merging**